lib/gibbon/api_request.rb

```diff
@@ -134,6 +134,9 @@ def rest_client
       client = Faraday.new(self.api_url, proxy: self.proxy) do |faraday|
         faraday.response :raise_error
         faraday.adapter adapter
+        if @request_builder.debug
```

Comment:
Note: Not sure if you want the debug attribute to be included as one of the "convenience accessors". It is only used here.